src/bp/ui-studio/src/web/components/Content/Select/index.tsx

```diff
@@ -16,9 +17,10 @@ import Loading from '~/components/Util/Loading'
 import { CONTENT_TYPES_MEDIA } from '~/util/ContentDeletion'
 
 import withLanguage from '../../Util/withLanguage'
+import parentStyle from '../style.scss'
```

Comment:
I see again alert button and modal from bootstrap imported im this file, is this necessary ?

Reply:
Since it's to fix an issue, I'd rather avoid going too deep into refactoring, it will most likely change in the near future too anyway.